ReSharper.FSharp/test/data/features/quickFixes/removeUnusedNamedPattern/Unused record named pattern with a single field.fs.gold

```diff
@@ -1,3 +1,3 @@
 type R = { F1: int }
 match { F1 = 1 } with
-| _{caret} -> ()
+| { F1 = _{caret} } -> ()
```

Comment:
This change looks unexpected: there's another quick fix that does it, and it's safe to remove the record pattern in this case.

Reply:
When removing the last record field pattern, like in this test.